### PR TITLE
Allow mounting components directly to an HTMLElement instead of a CSS selector

### PR DESCRIPTION
**Pull request by keichinger**
| Q             | A
| ------------- | --------------------------------------------------------------------- |
| Bug fix?      |no                                                                |
| New feature?  | yes                   |
| Improvement?  |no     |
| BC breaks?    |no                                                                |
| Deprecations? |no     |
| Docs PR       | **missing**                                   |

### CODE REVIEW

**Comment by keichinger** [comment]
Currently the CI fails because Browserstack is having its issues again where we're testing with 7 agents but only 6 report back, therefore it's marked as failed.

File: mount/index.ts [hunk 1]
```diff
@@ -8,9 +8,16 @@ import {parseElementAsJson} from "../json";
 /**
  * Mounts a Preact function or class component into all elements matching the given selector.
  */
-export function mountJsx<TPreactComponent extends ComponentFactory<any>>(selector: string, mountable: TPreactComponent, options?: mojave.ComponentMountOptions<TPreactComponent>): void
+export function mountJsx<TPreactComponent extends ComponentFactory<any>>(selector: string|HTMLElement, mountable: TPreactComponent, options?: mojave.ComponentMountOptions<TPreactComponent>): void
```
Comment (apfelbox) [inline mount/index.ts]:
```suggestion
export function mountJsx<TPreactComponent extends ComponentFactory<any>>(selector: string|HTMLElement[], mountable: TPreactComponent, options?: mojave.ComponentMountOptions<TPreactComponent>): void
```

File: mount/index.ts [hunk 2]
```diff
@@ -8,9 +8,16 @@ import {parseElementAsJson} from "../json";
...
 {
-    find(selector).forEach(node => doMountJsx<TPreactComponent>(node, mountable, options));
```
Comment (apfelbox) [inline mount/index.ts]:
```js
let elements = typeof selector == "string" ? find(selector) : selector;
elements.forEach(node => doMountJsx<TPreactComponent>(node, mountable, options));
```

File: mount/index.ts [hunk 3]
```diff
@@ -22,9 +26,11 @@ export function mountJsx<TPreactComponent extends ComponentFactory<any>>(selecto
  *
  *     mountLazyJsx<MyPreactComp>(".selector", () => import("./src/MyPreactComp"));
  */
-export function mountLazyJsx<TPreactComponent extends ComponentFactory<any>>(selector: string, importer: () => Promise<any>, options?: mojave.ComponentMountOptions<TPreactComponent>) : void
+export function mountLazyJsx<TPreactComponent extends ComponentFactory<any>>(selector: string|HTMLElement, importer: () => Promise<any>, options?: mojave.ComponentMountOptions<TPreactComponent>) : void
```
Comment (apfelbox) [inline mount/index.ts]: Use `HTMLElement[]` here as well.

Comment (keichinger) [reply mount/index.ts]: Oh, that one must've slipped through.

**Pull request merged**